modules/preloader/mistUI.js

```diff
@@ -5,7 +5,7 @@
 require('./include/common')('mist');
 const { ipcRenderer: ipc, remote, webFrame } = require('electron');
 const { Menu, MenuItem } = remote;
-const syncDb = require('../syncDb.js');
+const dbSync = require('../dbSync.js');
```

Comment:
Its just a name refactor. The reason is that now the files db.js and dbSync.js are next to each other, which helps with mental grouping.
Also we calle the nodeSync: nodeSync ;) so consistency.

This has actually northie todo with the vu.